packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -175,43 +175,66 @@ export class AppData extends DataObject {
 	}
 
 	private populateSharedTree(sharedTree: ITree): void {
-		// Set up SharedTree for visualization
-		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
-
-		// TODO: Maybe include example handle
+		const builder = new SchemaFactory("TodoList_Schema");
+
+		class TodoItem extends builder.object("todo-item", {
+			title: builder.string,
+			completed: builder.boolean,
+			dueDate: builder.optional(builder.string),
+			assignee: builder.optional(builder.string),
+			colloborators: builder.optional(builder.array(builder.string)),
+		}) {}
 
-		class LeafSchema extends builder.object("leaf-item", {
-			leafField: [builder.boolean, builder.handle, builder.string],
+		class TodoObject extends builder.object("todo-list", {
+			items: builder.array(TodoItem),
 		}) {}
 
-		class ChildSchema extends builder.object("child-item", {
-			childField: [builder.string, builder.boolean],
-			childData: builder.optional(LeafSchema),
+		class TodoCategory extends builder.object("todo-category", {
```

Comment:
Nit: could this be a map node? I could see that making sense for an app like this (allow users to create their own custom lists), and it would let us test the visuals of map nodes.

Reply:
```typescript
class TodoObject extends builder.object("todo-list", {
	items: builder.array(TodoItem),
}) {}

const TodoCategory = builder.map("todo-category", [TodoObject]);

class TodoWorkspace extends builder.object("todo-workspace", {
	lists: TodoCategory,
}) {}
```

Changed as below.